docker-compose.yml

```diff
@@ -18,60 +16,15 @@ services:
     build:
       context: .
       dockerfile: ./docker-server.Dockerfile
-      args:
-        APP: server
       <<:
         x-bake:
           platforms:
             - linux/arm64
             - linux/amd64
-    image: universalconnectfoundation/ucw-app-server:v${DOCKER_IMAGE_SERVER}
     networks:
       - ucw
     ports:
       - "${SERVER_PORT}:${SERVER_PORT}"
-    env_file:
-      - ./apps/server/.env
```

Comment:
If you remove this, how are you accessing the end-user's env file?

Reply:
The env file gets copied over with the rest of the code and loaded through dotenv